script.js

```diff
@@ -1,12 +1,363 @@
+  cardDiv.appendChild(additionalDetails);
+
+  layoutSelector.appendChild(cardDiv);
+  return cardDiv;
+}
+
+function createEpisodeCard(episode) {
+  const layoutSelector = document.querySelector(".layout");
+
+  const linkCardsToTVMaze = createElement("a", "");
+  linkCardsToTVMaze.setAttribute("href", episode.url);
+  linkCardsToTVMaze.setAttribute("target", "_blank");
+
+  const cardDiv = createElement("section", "episode__card");
+
+  const seasonAndEpisodeWithPadding = formatSeasonAndEpisode(
+    episode.season,
+    episode.number
+  );
+  const filmTitleElement = createElement("h1", "episode__card__title");
+  filmTitleElement.textContent = `${episode.name}`;
+  if (episode.season) {
+    filmTitleElement.textContent = `${episode.name} - ${seasonAndEpisodeWithPadding}`;
+  }
+
+  const filmImgElement = createElement("img", "episode__card__img");
+  if (episode.image && episode.image.original) {
+    filmImgElement.setAttribute("src", episode.image.original);
+  } else {
+    filmImgElement.setAttribute("src", "images/no-img-available.jpeg");
+    filmImgElement.classList.add("no_img_available");
+  }
+
+  const filmSummaryElement = createElement("p", "card__summary");
+  filmSummaryElement.innerHTML = episode.summary;
+
+  layoutSelector.appendChild(linkCardsToTVMaze);
+  linkCardsToTVMaze.appendChild(cardDiv);
+  cardDiv.appendChild(filmTitleElement);
+  cardDiv.appendChild(filmImgElement);
+  cardDiv.appendChild(filmSummaryElement);
+
+  return linkCardsToTVMaze;
+}
+
+async function searchShowsDropDown() {
+  try {
+    const showsData = await getAllShows();
+    const defaultOptionShow = document.createElement("option");
+    defaultOptionShow.textContent = "Select show...";
+    defaultOptionShow.setAttribute("value", "default");
+    showDropDown.appendChild(defaultOptionShow);
+
+    showsData.forEach((show) => {
+      const listOfShows = document.createElement("option");
+      listOfShows.textContent = `${show.name}`;
+      showDropDown.appendChild(listOfShows);
+    });
+
+    showDropDown.addEventListener("change", async function (event) {
+      if (event.target.value === "default") {
+        searchField.setAttribute("placeholder", "Search for show/s...");
+        searchField.removeEventListener("input", searchFilterEpisodes);
+        searchField.addEventListener("input", searchFilterShows);
+        searchField.value = "";
+        displayingFilterResults.textContent = "";
+        removeCardLayout();
+        clearCards();
+        render();
+        return;
+      } else {
+        searchField.setAttribute("placeholder", "Search for episode/s...");
+        const findID = showsData.find(
+          (show) => show.name === event.target.value
+        );
+        currentState.showsID = findID.id;
+        currentState.allEpisodes = await getAllEpisodes();
+        addCardLayout();
+        clearCards();
+        makePageForEpisodes(currentState.allEpisodes);
+        currentState.episodesFetched = false;
```

Comment:
Why is `currentState.episodesFetched` set to `false` after the episodes are being fetched?